src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -469,7 +469,7 @@ function loadOperation(
         Path: fullPath,
         ExternalDocsUrl: externalDocs?.url,
         RequestMediaTypes: mediaTypes.length > 0 ? mediaTypes : undefined,
-        BufferResponse: false,
+        BufferResponse: true,
```

Comment:
why is this change?

Reply:
To fix https://github.com/Azure/autorest.csharp/issues/2625